dask/bytes/local.py

```diff
@@ -15,6 +17,12 @@
 logger = logging.getLogger(__name__)
 
 
+def write_bytes(values, paths, compression=None, encoding=None):
+    """ See dask.bytes.core.write_bytes for docstring """
+    out = [delayed(open)(path, 'wb') for path in paths]
+    return out
```

Comment:
This doesn't appear to use values at all.  Instead it looks like this function is supposed to return a file-like object.  Perhaps we should change the name/signature?